CHANGELOG.md

```diff
@@ -14,6 +14,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   solar noon via the EXAMPLE OF USE in the subroutine header. See DESCRIPTION in code for more
   details. Provides the TRUE local solar hour angle (i.e., with equation of time included), but
   can also provide the MEAN value (without EOT) via FORCE_MLSHA=.TRUE. optional argument.
+- Changed call to MAPL_SunOrbitCreate() inside MAPL_Generic.F90 to new MAPL_SunOrbitCreateFromConfig,
+  the latter which get the orbital parameters from the MAPL state's Config. In this way no default
+  orbital parameter values need appear in MAPL_Generic.F90. Rather, these default values are encap-
+  sulated where they belong in Sun_Mod in base/MAPL_sun_uc.F90 and are now explicitly named and
+  commented at the head of the module. This is a structural zero-diff change.
```

Comment:
This should be moved into the `CHANGED` section.

Reply:
@tclune done.